docs/guidelines/content-guidelines/10-docs-preview.md

```diff
@@ -1,51 +1,26 @@
+## Steps
 
-Previews are built for PRs containing changes made in all the mentioned paths within these repositories for both successful and failed builds. The only exception is the `kyma` repository. [Netlify](https://www.netlify.com/) publishes notifications only for successful builds on such PRs.
+To preview content on the Kyma website, save your changes and run the local server.
 
-## Access previews
+1. Run `docsify serve docs` in the `/kyma` repository.
+2. Preview `https://kyma-project.io` in your browser on http://localhost:3000.
 
-After you create a PR, Netlify attaches a notification to your PR on the build status and a link to:
+## Preview module documentation
 
-- A website preview for successful builds
-
-    ![Successful preview](./assets/successful-preview.png)
-
-    >**NOTE:** The preview rebuilds automatically under the same link after every commit.
-
-- A direct link to log details for failed builds
-
-    ![Failed preview](./assets/failed-preview.png)
-
-The website preview deployment is also visible on the list of jobs that need to pass for the PR before it can be merged. Click on **Details** to go to the build logs.
-
-![Job details](./assets/job-details.png)
-
->**NOTE:** For the `kyma` repository, Netlify only attaches notifications for successful preview builds. Thus, if you make any changes in the `/docs` folder and there is no website preview attached to your PR, access failed build information through the link under **Details** available next to the preview job.
-
-## Add the preview to a repository
-
-1. Make sure that the documentation of the given repository is rendered on the [Kyma website](https://kyma-project.io).
-
-2. Contact the maintainers of the [Kyma website](https://kyma-project.io) to configure [Netlify](https://www.netlify.com/) for the new repository.
-
-3. Copy the [`build-preview.sh`](https://github.com/kyma-project/cli/blob/main/.kyma-project-io/build-preview.sh) script to the `.kyma-project-io` folder in the new repository. In this script, change the name of the repository under the exported **APP_PREPARE_FOR_REPO** environment:
-
-    ```bash
-    export APP_PREPARE_FOR_REPO="{REPOSITORY_NAME}"
-    ```
-
-4. Merge changes and [Netlify](https://www.netlify.com/) will start generating website previews on PRs in the new repository.
+1. In your module repository, create a pull request with documentation changes.
+2. In the `/kyma` repository, go to the [`docs/index.html`](https://github.com/kyma-project/kyma/blob/main/docs/index.html) file and change the value of the **alias** parameter for your module. By default the value points to the raw version of the `/docs/user` folder on the main branch of your module repository. Change the value to point to the raw version of the respective folder in your pull request or on your branch.
```

Comment:
```suggestion
2. In the `/kyma` repository, go to the [`docs/index.html`](https://github.com/kyma-project/kyma/blob/main/docs/index.html) file and change the value of the **alias** parameter for your module. By default, the value points to the raw version of the `/docs` folder on the main branch of your module repository. Change the value to point to the raw version of the respective folder on your branch.
```
I'd say all aliases are set to point to the `/docs` folder in our module repos. If you want to preview changes from your pull request, you must point to the branch and path in your forked repo that contains the changes:

```html
alias: {
'/{MODULE_REPOSITORY_NAME}/(.*)': 'https://raw.githubusercontent.com/{YOUR_GITHUB_USERNAME}/{FORKED_MODULE_REPOSITORY_NAME}/{BRANCH_TO_PREVIEW}/docs/$1',
}, 
```
For example:
```html
alias: {
'/istio/(.*)': 'https://raw.githubusercontent.com/kymauser/istio/docu-update/docs/$1',
}, 
```
I'm not sure if you can also point to the pull request